doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+  ``SSDB`` environment variables per orchestrator with the ``db_identifier`` argument suffixed. In this example, the created env vars are
+  ``SSDB_single_shard_db_identifier`` and ``SSDB_clustered_db_identifier``. The environment variables
+  hold the address of the associated database. The two environment variables will be used later in the example
+  to establish a SmartRedis ``Client`` connection with the launched databases.
+
+Next within the workflow, we send tensors to the launched databases with the ``Client`` SmartRedis API.
+
+The SmartRedis ``Client`` object contains functions that manipulate, send, and receive
+data within the database. Each database has a single, dedicated SmartRedis ``Client``.
+Begin by initializing a SmartRedis ``Client`` object per launched database.
+
+To create a designated SmartRedis ``Client``, you need to specify the address of the target
+running database. You can easily retrieve this address using the ``get_address()`` function
+provided by the initialized ``Orchestrator`` object.
+
+.. note::
+  Make sure to set the ``cluster`` argument appropriately based on whether the database is in a clustered configuration or not.
+
+Here's the code example:
+
+.. code-block:: python
+
+  sr_client_colo = Client(cluster=False, address=colocated_orch.get_address()[0], logger_name="Colo client logger")
+  sr_client_clus = Client(cluster=True, address=clustered_orch.get_address()[0], logger_name="Clus client logger")
+
+In this code, ``sr_client_colo`` is an initialized client for the colocated Orchestrator, while ``sr_client_clus`` is an
+initialized client for the clustered Orchestrator.
+
+In this example, we transmit, retrieve and manipulate a NumPy tensor. Begin by creating a
+NumPy array and send it to each database using the assigned SmartRedis client. To
+accomplish this, use the ``put_tensor()`` function provided by the SmartRedis ``Client`` API.
+We access this function through the SmartRedis ``Client`` objects, ``sr_client_colo`` and ``sr_client_clus``.
+In this example, we are sending different numpy arrays to both databases.
+
+.. code-block:: python
+
+  array_colo = np.array([1, 2, 3, 4])
+  sr_client_colo.put_tensor("tensor_1", array_colo)
+
+  array_clus = np.array([5, 6, 7, 8])
+  sr_client_clus.put_tensor("tensor_2", array_clus)
+
+Lets check to make sure the database tensors are not in the opposite databases
+
+.. code-block:: python
+
+  check_colo_db_tensor_incorrect = sr_client_colo.key_exists("tensor_2")
+  check_clus_db_tensor_incorrect = sr_client_clus.key_exists("tensor_1")
+  logger.info(f"The colo array key exists in the incorrect database: {check_colo_db_tensor_incorrect}")
+  logger.info(f"The clus array key exists in the incorrect database: {check_clus_db_tensor_incorrect}")
+
+When you execute the program the following output will appear.
+
+.. code-block:: bash
+
+  21:01:54 osprey.us.cray.com SmartSim[105901] INFO The colo array key exists in the incorrect database: False
```

Comment:
Globally: replace osprey.us.cray.com with something generic like "system.host.com". I'm not sure what that 105901 is, but consider replacing it as well